challenge_2/csharp/fojo/Challenge2.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Linq;
+
+namespace ConsoleApplication
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // var array = new int[] { 2, 3, 4, 2, 3, 5, 4, 6, 4, 6, 9, 10, 9, 8, 7, 8, 10, 7 };
```

Comment:
Sorry for the late reply. Are you referring to removing the commented-out code, or writing test suites? If you can clarify which part is blocking you I'm happy to help you out.